packages/admin/src/form/FinalFormRangeSlider.tsx

```diff
@@ -0,0 +1,135 @@
+import { FormControl, Slider, SliderProps, Theme, WithStyles } from "@material-ui/core";
+import { createStyles, withStyles } from "@material-ui/styles";
+import * as React from "react";
+import { FieldRenderProps } from "react-final-form";
+
+import { InputBase } from "./InputBase";
+
+export type CometAdminFinalFormRangeSliderClassKeys =
+    | "root"
+    | "inputsWrapper"
+    | "inputFieldsSeperatorContainer"
+    | "sliderWrapper"
+    | "inputFieldContainer";
+
+const styles = (theme: Theme) => {
+    return createStyles<CometAdminFinalFormRangeSliderClassKeys, any>({
+        root: {
+            padding: "0 20px",
+            width: "100%",
+        },
+        inputsWrapper: {
+            justifyContent: "space-between",
+            marginBottom: "15px",
+            alignItems: "center",
+            display: "flex",
+        },
+        inputFieldsSeperatorContainer: {
+            textAlign: "center",
+            minWidth: "20%",
+        },
+        sliderWrapper: {
+            paddingBottom: "20px",
+        },
+        inputFieldContainer: {
+            textAlign: "center",
+        },
+    });
+};
+
+interface IFinalFormRangeSliderProps extends FieldRenderProps<any, HTMLInputElement> {
```

Comment:
FieldValue parameter (any) should be properly typed to { min: number, max: number }